Signum.Engine.Extensions/Workflow/WorkflowNodeGraph.cs

```diff
@@ -238,6 +238,19 @@ public void Validate(List<WorkflowIssue> issuesContainer, Action<WorkflowGateway
 
                 if (e.Type == WorkflowEventType.IntermediateTimer && !e.Name.HasText())
                     issues.AddError(e, WorkflowValidationMessage.IntermediateTimer0ShouldHaveName.NiceToString(e));
+
+                if (e.Type == WorkflowEventType.BoundaryInterruptingTimer)
+                {
+                    var parentActivity = Activities.Values.Where(a => a.BoundaryTimers.Contains(e)).SingleEx();
+                    if (string.IsNullOrWhiteSpace(e.DecisionOptionName) && parentActivity.Type == WorkflowActivityType.Decision)

```

Comment:
The new field becomes now mandatory for InterruptingTimers on Decision activities. Right?

Reply:
yes